### DISCUSSION TITLE
GPU accelerated Solving

### DISCUSSION THREAD

**Discussion by okkarkyaw122** [OpenRadioss/OpenRadioss#3133] [Q&A] [answered]
Hello, can I do GPU accelerated solving with OpenRadioss?
Thank you.

**Comment by MarianBulla** [comment, marked as answer]
Hello @okkarkyaw122, 
thanks for your interest in OpenRadioss.
Currently we are not supporting GPU solving with OpenRadioss.
It is because OpenRadioss is an industrial crash solver, developed several decades, ago.
Re-writing it to GPU would take an extremly high effort.
Anyway, we are open for these efforts and will support everyone, who is willing to collaborate and contribute into this task.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by okkarkyaw122** [reply]
Thank you so much for the answer. That helps a lot in choosing a new PC.
